### PR TITLE
doc/releases: add Nautilus 14.2.12 release notes

### PR DESCRIPTION
**Pull request by smithfarm**
Also fix missing Octopus 15.2.5 in "doc/releases/general.rst" and
"doc/releases/releases.yml"

Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by theanalyst** [comment]
jenkins render docs

**Comment by smithfarm** [comment]
For me, https://docs.ceph.com/ceph-prs/37718/ shows "502 Bad Gateway"

**Comment by neha-ojha** [comment]
https://ceph--37718.org.readthedocs.build/en/37718/ - the rendering is available with `docs/readthedocs.org:ceph` check under 7 checks

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,153 @@
...
+  to be warned again if additional repairs are performed you can provide a value
+  to the command and specify the value of ``mon_osd_warn_num_repaired``.
+  This command will be replaced in future releases by the health mute/unmute feature.
+
+* It is now possible to specify the initial monitor to contact for Ceph tools
+  and daemons using the ``mon_host_override`` config option or
+  ``--mon-host-override <ip>`` command-line switch. This generally should only
+  be used for debugging and only affects initial communication with Ceph's
+  monitor cluster.
+
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst:27]: Can we please add a note similar this https://github.com/ceph/ceph/pull/37109/files#diff-52fc1d32fab56fed8f65b5e2db4f9ea3e3e7240f4b7797ef2b1a0b6dcb5c3a1bR25-R27 and reference https://tracker.ceph.com/issues/47296 and https://github.com/ceph/ceph/pull/36982 instead

**Review by neha-ojha — APPROVED** [review]
looks good with the comment addressed

**Comment by smithfarm** [comment]
@djgalloway With regard to comment https://github.com/ceph/ceph/pull/37718#issuecomment-713146199 it would appear that Jenkins now replies to "jenkins render docs" with a bogus URL. Would it make sense to quash this confusing reply that Jenkins is posting?

File: doc/releases/nautilus.rst (hunk 1 above)
Comment (smithfarm) [reply doc/releases/nautilus.rst:27]: Thanks, @neha-ojha - I have made the change you requested.

**Comment by djgalloway** [comment]
> @djgalloway With regard to comment [#37718 (comment)](https://github.com/ceph/ceph/pull/37718#issuecomment-713146199) it would appear that Jenkins now replies to "jenkins render docs" with a bogus URL. Would it make sense to quash this confusing reply that Jenkins is posting?

https://github.com/ceph/ceph-build/pull/1687

File: doc/releases/nautilus.rst (hunk 1 above)
Comment (neha-ojha) [reply doc/releases/nautilus.rst:27]: Thanks @smithfarm!

**Comment by smithfarm** [comment]
jenkins test make check

**Review by jdurgin — APPROVED** [review]

**Pull request merged**